apps/mobile/src/utils/transaction.ts

```diff
@@ -30,10 +36,19 @@ export const createAndSend: CreateAndSendFunction = async (
 		}
 
 		try {
-			res.signatureString = await solanaHandlers.signAndSendTransaction(
-				transaction,
-				privateKey,
-			);
+			if (payload.tokenForFee.metadata?.symbol === 'SOL') {
+				res.signatureString = await solanaHandlers.signAndSendTransaction(
+					transaction,
+					privateKey,
+				);
+			} else {
+				res.signatureString =
+					await solanaHandlers.signTransactionAbstractionFee(
+						transaction,
+						privateKey,
+					);
```

Comment:
does it sign and also send the transaction?

Reply:
yes, should i change the name?